frontend/src/app/main/ui/shapes/custom_stroke.cljs

```diff
@@ -401,7 +401,7 @@
         stroke-props (-> (obj/new)
                          (obj/set! "id" (dm/fmt "strokes-%" (:id shape)))
                          (cond->
-                          (some? (:blur shape))
+                          (and (some? (:blur shape)) (not= :frame (:type shape)))
```

Comment:
I think we should use here the `app.common.page.helpers/frame-shape?` predicate.